src/Motion.cpp

```diff
@@ -128,6 +128,8 @@ void Motion::detect()
 
 int Motion::init()
 {
+    int i = 0;
```

Comment:
Why not just define it within the for loop?

Reply:
It's been a long time since I programmed in C/Cpp... Just old habits. I'll change it...